assets/js/atomic/blocks/product-elements/average-rating/block.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "woocommerce/product-average-rating",
+	"version": "1.0.0",
+	"title": "Product Average Rating",
+	"description": "Display the average rating of a product",
+	"attributes": {
+		"textAlign": {
+			"type": "string",
+			"default": ""
```

Comment:
Nitpicking: Perhaps we can remove the default value for this since it seems we don't intent to set any default value here.